py4web/core.py

```diff
@@ -600,8 +600,7 @@ def URL(
     URL('a','b',vars=dict(x=1),use_appname=False) -> /{script_name?}/a/b?x=1
     """
     if use_appname is None:
-        use_appname = request.headers.get("x-py4web-appname")
-        use_appname = True if use_appname is None else not use_appname
+        use_appname = request.headers.get("x-py4web-appname", "") == ""
```

Comment:
Fine!

Reply:
Your version sounds good
use_appname = not request.headers.get("x-py4web-appname")